src/layouts/error.vue

```diff
@@ -1,30 +1,116 @@
 <template>
-  <div class="grid-container full">
-    <main class="not-found">
-      <h1 v-if="error.statusCode === 404">
-        {{ $t('error.not-found') }}
+  <main class="bg-yellow h-screen relative page-404">
+    <Logo class="pt-6 lg:pt-8 ms-6 lg:ms-10 w-30 h-auto" />
```

Comment:
Shouldn't the logo be a link to home?

Reply:
And it needs something for a11y, or maybe aria-hidden if it's not a link?